Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,7 +101,16 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            if (!IsNullOrEmpty(array) && IsIndexIsWithinRange(array, "Remove", index))
+            {
+                var shortenedArray = new int[array.Length - 1]; ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)